catalog/transformer/catalog-transformer-html/src/main/java/ddf/catalog/transformer/html/models/HtmlCategoryModel.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.html.models;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import org.apache.commons.lang.WordUtils;
+
+public class HtmlCategoryModel implements HtmlExportCategory {
+
+  private String title;
+
+  private List<String> attributeList;
+
+  private Map<String, Object> attributeMappings;
```

Comment:
nit - Would prefer `Map<String, HtmlValueModel>`

Reply:
Good catch! That was a remnant of the prototype